app/content/power/contract-inventory.tsx

```diff
@@ -6,29 +6,28 @@ import { Resources } from "~/components/resources/Resources";
 const info = {
   name: "contract-inventory",
   path: "/power/contract-inventory",
-  title: "Contract Inventory",
-  shortname: "Contract Inventory",
+  title: "Game Inventory",
+  shortname: "Game Inventory",
+  platforms: {
+    unreal: "https://docs.sequence.xyz/sdk/unreal/introduction",
+    unity: "https://docs.sequence.xyz/sdk/unity/overview",
+    telegram: "https://docs.sequence.xyz/guides/telegram-integration",
+    web: "https://docs.sequence.xyz/solutions/wallets/sequence-kit/getting-started",
+  },
   image: {
     src: "contract-inventory",
   },
   description:
-    "A simple and elegant way to view your wallet inventory for a specific contract, with Sequence Kit",
+    "Simplify the experience for your users by showing them only the assets relevant to your game or app.",
 } as const;
 
 function component() {
   const { address } = useAccount();
 
   return (
     <>
-      Sometimes a player just wants to see the items they have in the current
-      game.
-      <br />
-      <br />
-      It should be as simple as it sounds.
-      <br />
-      <br />
-      With Sequence Kit, it is!
       <h2>View Assets from a Single Contract</h2>
+      We combine our Indexer for realtime querying with Sequence Kit to simplify this common flow.
```

Comment:
Alternative:
"Sequence Kit combines our Indexer with elegant out-of-the-box UI, to simplify realtime inventory viewing and management."